### DISCUSSION TITLE
Can't install StabilityMatrix on Mint (Ubuntu)

### DISCUSSION THREAD

**Discussion by goffy76** [LykosAI/StabilityMatrix#933] [Q&A] [answered]
I'm completely new to Linux, and I want to run Stabel Diffusion, but I can't make it work, so I want to try StabilityMatrix, that's not working either :-/ I keep getting this massage:

artin@Martin-Linux:~$ sudo apt install https://github.com/LykosAI/StabilityMatrix.git
[sudo] password for martin:     
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
E: Unable to locate package https://github.com/LykosAI
martin@Martin-Linux:~$ 

What the H... am I doing wrong?

Regards Martin

**Comment by MichT368** [comment, marked as answer]
Skip the automatic installing of repos during the install process completely to get a "naked" installation. Then go to the plus-symbol at the botton and choose you model there and continue.

**Comment by airtonix** [comment]
@goffy76 New to linux? 

1. [go to the latest release](https://github.com/LykosAI/StabilityMatrix/releases/latest) 
2. download the linux zip file
3. unzip it
4. right click the `.appimage` > properties > Executable as program: ✅

![image](https://github.com/user-attachments/assets/dc5f53e2-dba8-4c2b-ae36-14d014604c03)

Then, for bonus points: 

install menulibre 
  a. press `win`/`mac`/`super` key
  b. type `Software`, launch it.
  c. search for `Menu editor` 
  d. install it.

Create a new menu launcher

![image](https://github.com/user-attachments/assets/915e5bd9-d15f-4919-a2ae-2751b6c9a2ab)

![image](https://github.com/user-attachments/assets/0f45c916-1563-4608-8e48-c3e0384b91a1)


Then wait a bit (maybe logout and in again), then You should now have a Stability Matrix launcher.

![image](https://github.com/user-attachments/assets/ded4c224-276f-4365-b5d5-d1838b7e81e5)
